components/file/aliyun/file.go

```diff
@@ -14,46 +14,40 @@
  * limitations under the License.
  */
 
-package alicloud
+package aliyun
```

Comment:
`aliyun` is a chinese name, while `alicloud` is english.
I propose that we keep the package name `alicloud` and name the component as `alicloud.oss`, which is consistent with other components' name.
See https://docs.dapr.io/zh-hans/reference/components-reference/supported-bindings/alicloudoss/

![image](https://user-images.githubusercontent.com/26001097/179887548-46ad0a25-f042-40ff-a677-97203a32a361.png)

Reply:
![image](https://user-images.githubusercontent.com/11661906/179914290-0d964280-19c3-454c-ba0a-e9aeb40833b4.png)

https://cloudinit.readthedocs.io/en/latest/topics/datasources/aliyun.html